Dockerfile

```diff
@@ -4,21 +4,38 @@ ENV GSPLOT_VERSION_DEFAULT=0.8.1
 
 COPY inst/extdata/installPackages.R /tmp/install/installPackages.R
 
-# Get GSPlot dependency installs
-RUN mkdir -p /tmp/install/gsplot_description_dir && \
-  wget -O /tmp/install/gsplot_description_dir/DESCRIPTION https://raw.githubusercontent.com/USGS-R/gsplot/v${GSPLOT_VERSION:-$GSPLOT_VERSION_DEFAULT}/DESCRIPTION
+RUN mkdir -p /tmp/install/gsplot_description_dir
+RUN mkdir -p /tmp/install/gsplot_source_dir
 
-# Install GSplot
+# Get GSPlot Sources
+RUN if getent ahosts "sslhelp.doi.net" > /dev/null 2>&1; then \
+    wget 'http://sslhelp.doi.net/docs/DOIRootCA2.cer' && \
+    wget --ca-certificate=DOIRootCA2.cer -O /tmp/install/gsplot_description_dir/DESCRIPTION https://raw.githubusercontent.com/USGS-R/gsplot/v${GSPLOT_VERSION:-$GSPLOT_VERSION_DEFAULT}/DESCRIPTION && \
+    wget --ca-certificate=DOIRootCA2.cer -O /tmp/install/gsplot_source_dir/gsplot.zip https://github.com/USGS-R/gsplot/archive/v${GSPLOT_VERSION:-$GSPLOT_VERSION_DEFAULT}.zip ; \
+  else \
+    wget -O /tmp/install/gsplot_description_dir/DESCRIPTION https://raw.githubusercontent.com/USGS-R/gsplot/v${GSPLOT_VERSION:-$GSPLOT_VERSION_DEFAULT}/DESCRIPTION && \
+    wget -O /tmp/install/gsplot_source_dir/gsplot.zip https://github.com/USGS-R/gsplot/archive/v${GSPLOT_VERSION:-$GSPLOT_VERSION_DEFAULT}.zip ; \
+  fi ;
+  
+# Install GSPlot Dependencies
 RUN mkdir ${RSERVE_HOME}/R_libs && \
   mkdir ${RSERVE_HOME}/work && \
   cd /tmp/install/gsplot_description_dir && \
-  Rscript /tmp/install/installPackages.R && \
-  Rscript -e "library(devtools);install_url('https://github.com/USGS-R/gsplot/archive/v${GSPLOT_VERSION:-$GSPLOT_VERSION_DEFAULT}.zip', dependencies = F)"
+  Rscript /tmp/install/installPackages.R
 
 # Install Repgen Dependencies
 COPY DESCRIPTION /tmp/install/repgen/DESCRIPTION
 RUN cd /tmp/install/repgen && Rscript /tmp/install/installPackages.R
 
+# Build and install GSPlot Package
```

Comment:
Replaced the `"...install_url(...)..."` version of the GSPlot install with a manual version that works the same way because I couldn't figure out how to inject the cert properly for the R `install_url` command. This accomplishes the same thing as that command does, just externally from R (so that I could use the root cert when needed).